worker/backend/backend.go

```diff
@@ -148,7 +216,20 @@ func (b *Backend) BulkMetrics(handles []string) (metrics map[string]garden.Conta
 //
 // Errors:
 // * Container not found.
-func (b *Backend) Lookup(handle string) (container garden.Container, err error) { return }
+func (b *Backend) Lookup(handle string) (garden.Container, error) {
+	ctx := namespaces.WithNamespace(context.Background(), b.namespace)
```

Comment:
Does it make sense for this to be a context.TODO as we intend it to be replaced ?

Reply:
yes